xdsl/dialects/riscv.py

```diff
@@ -2491,6 +2491,7 @@ class LiOp(RISCVInstruction, ABC):
 
     rd = result_def(IntRegisterType)
     immediate = attr_def(base(Imm32Attr) | base(LabelAttr))
+    fastmath = opt_attr_def(FastMathFlagsAttr)
```

Comment:
Is there a specific reason this is an attribute here where it was a property on the other PR?

Reply:
I would recommend changing to a property here